tests/sqlparser_duckdb.rs

```diff
@@ -246,6 +246,67 @@ fn test_create_table_macro() {
     assert_eq!(expected, macro_);
 }
 
+#[test]
+fn test_select_from_first() {
```

Comment:
@mitsuhiko could we move this test to [sqlparser_common.rs](https://github.com/apache/datafusion-sqlparser-rs/blob/8026966ba12b6cb7f2ee9bfa92d80e48ff1650f7/tests/sqlparser_common.rs) and use the `all_dialects_where(|d| d.supports_from_first_select())`? so that it covers both duckdb and clickhouse (and any future dialects)

Reply:
Moved it.